.gitignore

```diff
@@ -1,4 +1,5 @@
 client/node_modules
 client/dist/
+client/out/
```

Comment:
Out of curiosity, what are we producing in `out`?

Reply:
From the package.json it's for the compile tests. This is actually from the vscode defaults, at some point we should look into it more ?